cli/JSXV4UPGRADE.md

```diff
@@ -0,0 +1,44 @@
+**JSX V4 upgrade path**
+
+1. No upgrading rescript-react && no changes in bsconfig.json
+
+   JSX V3 by default.
```

Comment:
Unless we did any changes in the compiler, if there's no `react` setting in your `bsconfig`, no JSX transformation will be applied.

So either that sentence is wrong, or I missed the change in the compiler setting.

Reply:
Ah, I made a PR in the compiler for the new settings.